components/public-api-server/pkg/server/server.go

```diff
@@ -62,6 +62,9 @@ func Start(logger *logrus.Entry, version string, cfg *config.Configuration) erro
 		log.Info("No stripe webhook secret is configured, endpoints will return NotImplemented")
 	}
 
+	srv.HTTPMux().Handle("/test", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte(`test`))
```

Comment:
We'll remove this once we've got actual endpoints implemented, for now this acts as a quick check the server is reachable.